types/electra/attestation.yaml

```diff
@@ -0,0 +1,33 @@
+Electra:
+  IndexedAttestation:
+    type: object
+    description: "The [`IndexedAttestation`](https://github.com/ethereum/consensus-specs/blob/v1.5.0-alpha.2/specs/electra/beacon-chain.md#indexedattestation) object from the CL spec."
+    properties:
+      attesting_indices:
+        type: array
+        maxItems: 131072
+        description: "Attesting validator indices"
+        items:
+          $ref: "../primitive.yaml#/Uint64"
+      signature:
+        $ref: '../primitive.yaml#/Signature'
+        description: "The BLS signature of the `IndexedAttestation`, created by the validator of the attestation."
```

Comment:
What does "validator of the attestation mean"?

Reply:
IMU it means that it comes from the corresponding `Attestation` whose signature was created by the validator which generated it